### DISCUSSION TITLE
The Configuration Successful Window "The Client Configuration has been saved" showing everytime we try to open the seb file. How to ignore this window?

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#463] [Q&A]
Hi Team,

I have encrypted the .seb file using cryptographic identity.

![Screenshot from 2022-09-07 12-24-13](https://user-images.githubusercontent.com/111885140/188814497-c8cfa013-c746-494d-a304-bab93ff2b954.png)

But when I try to launch the .seb file, it shows the Configuration Successful window as "The Client configuration has been saved and will be used when you start SEB next time. Do you want to Quit for now?" and when we click "yes" the file closed, when we click "No" then only the SEB opened and show the login page of the exam which we given.

![Screenshot from 2022-09-07 12-24-36](https://user-images.githubusercontent.com/111885140/188814700-624ec1ec-2da4-4d45-9b68-d7aeea25283e.png)

How to ignore this window?

When candidate launch the SEB file during the exam, they will wait for login page only. But, if the candidates see this window, they may be confused and if they click Yes the SEB will be closed and then again they will open the SEB launch file and click No in the window. So, What is the option to ignore this window after launching the SE file? 

Kindly suggest.

**Comment by dbuechel** [comment]
> Use SEB settings file for ... starting an exam: A configuration file saved with this option will start the exam with the according settings, but won't change local SEB settings.

https://safeexambrowser.org/windows/win_usermanual_en.html#ConfigFilePane

**Reply by Chris251290** [reply]
Okay. 

Can we use the same encryption file on all other windows systems? If Yes, Will it change the Browser Exam Key at all other systems?

Kindly reply.

**Reply by dbuechel** [reply]
Yes, you certainly can use the same encryption settings for all Windows systems. And yes, the BEK is depending on the installed SEB version, so it may differ from one system to another (starting with version 3.4.0 also by build platform!), see https://safeexambrowser.org/windows/win_usermanual_en.html#ExamPane.

**Reply by Chris251290** [reply]
Hi Damian,

Thanks for yoiur reply.

Actually, We are going to use SEB Version 3.4.0 for upcoming Proctoring exam for all the candidates. We are going to create the exam using BEK. Then the same SEB file will be used to all the candidates.

 If the BEK may differ from one system to another after installed the same encryption file to all the candidates, How the exam login will show to all the candidates? How the candidates can able to attend the exam?

Kindly Clarify this.

**Reply by dbuechel** [reply]
If you are defining SEB 3.4.0 as the version to be used for your exam, then you'll need to register the BEK of the 32-bit as well as the 64-bit build of SEB 3.4.0 in your LMS / exam system. This is as easy as opening your exam configuration in the configuration tool of both builds of SEB 3.4.0 and copying the BEK respectively.

However, if you should happen to know that all of your students are using a 64-bit system (which is the case for the vast majority of modern systems), then you could only use the BEK generated by and for the 64-bit build and simply ignore the 32-bit part.
